packages/ckeditor5-media-embed/src/mediaregistry.ts

```diff
@@ -85,26 +86,25 @@ export default class MediaRegistry {
 	 *
 	 * **Note:** If no URL is specified, an empty view element is returned.
 	 *
-	 * @param {module:engine/view/downcastwriter~DowncastWriter} writer The view writer used to produce a view element.
-	 * @param {String} url The URL to be translated into a view element.
-	 * @param {Object} options
-	 * @param {String} [options.elementName]
-	 * @param {Boolean} [options.renderMediaPreview]
-	 * @param {Boolean} [options.renderForEditingView]
-	 * @returns {module:engine/view/element~Element}
+	 * @param writer The view writer used to produce a view element.
+	 * @param url The URL to be translated into a view element.
 	 */
-	getMediaViewElement( writer, url, options ) {
-		return this._getMedia( url ).getViewElement( writer, options );
+	public getMediaViewElement(
+		writer: DowncastWriter,
+		url: string,
+		options: MediaOptions
+	): ViewElement {
+		return this._getMedia( url )!.getViewElement( writer, options );
 	}
 
 	/**
 	 * Returns a `Media` instance for the given URL.
 	 *
 	 * @protected
-	 * @param {String} url The URL of the media.
-	 * @returns {module:media-embed/mediaregistry~Media|null} The `Media` instance or `null` when there is none.
+	 * @param url The URL of the media.
+	 * @returns The `Media` instance or `null` when there is none.
 	 */
-	_getMedia( url ) {
+	protected _getMedia( url: string ): Media | null {
```

Comment:
Is this method used anywhere in overriding class? If not maybe it should be `private`?